devito/operator.py

```diff
@@ -124,31 +123,32 @@ def __init__(self, expressions, **kwargs):
         # Finish instantiation
         super(Operator, self).__init__(self.name, nodes, 'int', parameters, ())
 
-    @cached_property
```

Comment:
Since this cannot be a ``cached_property`` anymore, we now incur a performance penalty at every operator invocation. Does this have an evident impact on the user experience?

Reply:
I think this is fairly ok, we incur a really small penalty compared to the actual run for non-tiny model. I also think it's better to have a few seconds extra deriving arguments rather than duplicating memory.

Reply:
"a few seconds" is a **HUGE** amount of time from the developer perspective! can you quantify the penalty? is it really seconds or just fractions of seconds? I guess you really meant fractions of seconds (ie imperceivable).

Reply:
It's like a few seconds for the full TRAVIS, so almost nothing individually.

Reply:
oh, OK then. I was talking in terms of individual operators.

Reply:
Hum, checking on `examples/seismic/acoustic/acoustic_example.py` it actually takes the exact same time  (on my machine). You losse on `cached_property` but you gain on skipping duplicate allocations